### PR TITLE
Sharding tests

### PR DESCRIPTION
**Pull request by senia-psm**

### CODE REVIEW

File: tests/context.rs [hunk 1]
```diff
@@ -60,24 +61,28 @@ impl EnvConfig {
...
+            n_worker_executors: shards,
         }
     }
+
+    pub fn from_env() -> EnvConfig {
+        Self::from_env_with_shards(3)
+    }
 }
 
 pub struct Context<'docker_client> {
```
Comment (afsalthaj) [inline tests/context.rs:73]: Always a bit of turn down for me to see connecting to docker from rust.

Comment (vigoo) [reply tests/context.rs:73]: Why?

Comment (senia-psm) [reply tests/context.rs:73]: It is for testcontainers

**Comment by senia-psm** [comment]
Closing in favour of #83

**Pull request closed without merging**